homework/content.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+(function(exports) {
+  
+	var ContentManager = function(){
+		this._wrapper = null;
+	};
+  
+	ContentManager.prototype = {
+
+		start() {
+			window.addEventListener('note-open', function(event) {
```

Comment:
Instead of binding an anonymous function, you should pass `this` as the listener of the API.